roles/splunk_common/tasks/add_forward_server.yml

```diff
@@ -14,20 +14,12 @@
     - "'forwarded-server already present' not in forward_status.stderr"
   no_log: "{{ hide_password }}"
 
-- name: "Adjust outputs.conf permissions"
```

Comment:
It looks like this was needed/added as part of our internal WIndows usage: https://github.com/splunk/splunk-ansible/pull/489

Reply:
yes I was container focused where we don't support ES/ITSI yet made adjustment

Reply:
Was there a workaround added for this fix? I'll kick off a windows ec2 test to verify as well.